README.md

```diff
@@ -41,6 +41,12 @@ To add a new pump.io account to p you need to authorize it:
 $ p authorize <webfinger>
 ```
 
+_example:_
+
+```
+$ p authorize my_user@mediagoblin.example.com
```

Comment:
"user@example.com" would be better IMO - p is a pump.io client not mediagoblin specific.